client/src/js/viruses/components/Detail/Schema.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+//import PropTypes from "prop-types";
+
+const Schema = () => (
```

Comment:
Save the schema code for another PR. Your bug fixes will be released sooner in a separate release from the schema feature.